bokehjs/test/models/transforms/jitter.coffee

```diff
@@ -43,6 +43,18 @@ describe "Jitter transform module", ->
       # We can set this deterministically because we've stubbed random
       expect(thediff).to.equal 0
 
+    it "should cache values for identical input lengths", ->
+      N = 100
+      val1 =  Array.apply(null, Array(N)).map ->
+                5
```

Comment:
Use `repeat(5, N)` (from `core/util/array`). Do similarly in other instances.

Reply:
Thanks, @mattpap!  I've made these changes.